File: contains-duplicate.java
class containsDuplicate {
    public static void main(String[] args) {
      containsDuplicate ins = new containsDuplicate();
      int[] arr = {1,2,3,1};
      System.out.println(ins.containsDuplicate(arr) == true);
      int[] arr2 = {1,2,3,4};
      System.out.println(ins.containsDuplicate(arr2) == false);
      int[] arr3 = {1,1,1,3,3,4,3,2,4,2};
      System.out.println(ins.containsDuplicate(arr3) == true);
      System.out.println("すべてTrueならコンプリート！");
    }

    public boolean containsDuplicate(int[] nums) {
        // ここにコードを書いてください
        return true;
    }
}

File: contains-duplicate.py
from typing import *

class Solution:
    def containsDuplicate(self, nums: List[int]) -> bool:
        # ここにコードを書いてください
        return False


print("スタート")
assert Solution().containsDuplicate([1,2,3,1]) == True
assert Solution().containsDuplicate([1,2,3,4]) == False
assert Solution().containsDuplicate([1,1,1,3,3,4,3,2,4,2]) == True
print("コンプリート!本番テストへ")
